ls1intum/Artemis
# Enhancement Guided Tour: Add tests and prefixes for tutorial selectors

### Checklist

### Motivation and Context
This PR covers a partial aspect of the issue https://github.com/ls1intum/Artemis/issues/1113

### Description
This PR includes a test for the `courseOverviewTour` and checks for every step if the highlight element is still available. Furthermore it adds prefixes to the tour selectors to enhance the maintainability of guided tutorials.

### Steps for Testing
This PR does not change any functionality. All guided tutorials (and Artemis use cases) should work as usual.
